phpstan.neon

```diff
@@ -22,3 +22,9 @@ parameters:
             message: "#^Parameter \\#1 \\$autoload_function of function spl_autoload_register expects callable\\(string\\)\\: void, \\$this\\(Roave\\\\BetterReflection\\\\Util\\\\Autoload\\\\ClassLoader\\) given\\.$#"
             count: 1
             path: src/Util/Autoload/ClassLoader.php
+        # The method does not exist in reflection
```

Comment:
We should probably assert that it is a `ReflectionNamedType`. Also to be checked: does this shrink the psalm baseline?